### PR TITLE
fix the expand directory when click file

### PR DESCRIPTION
**Pull request by icerove**
Associated Issue: #18 
Here's the contributing guide at https://github.com/wasdk/WebAssemblyStudio/wiki/Contributing

### Summary of Changes

* not expand all the directory when click the file

### Test Plan
run the command of ```yarn test``` and all the tests passed

### CODE REVIEW

**Comment by vgrichina** [comment]
@icerove still fails build:

```
ERROR in /builds/near-protocol/NEARStudio/src/monaco-utils.ts
[tsl] ERROR in /builds/near-protocol/NEARStudio/src/monaco-utils.ts(71,1)
      TS1128: Declaration or statement expected.
```

File: src/components/DirectoryTree.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ export class DirectoryTree extends React.Component<DirectoryTreeProps, {
       this.setState({ directory: props.directory });
     } else {
       this.tree.refresh();
-      MonacoUtils.expandTree(this.tree);
```
Comment (vgrichina) [inline src/components/DirectoryTree.tsx]: no need to comment out stuff, should just be removed

File: templates/01-hello-username/src/test.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 </head>
 <body style="background: #fff">
   <script src="https://cdn.jsdelivr.net/npm/js-cookie@2/src/js.cookie.min.js"></script>
-  <script src="https://cdn.jsdelivr.net/gh/nearprotocol/nearlib@09127f91aace8b95f97e8c51d5d32c5c7ad555e2/dist/nearlib.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/nearlib@0.14.1/dist/nearlib.js"></script>
```
Comment (vgrichina) [inline templates/01-hello-username/src/test.html]: shouldn't be in this PR

**Comment by vgrichina** [comment]
@icerove why did you remove `test.html` files? Are they auto-generated now?

**Comment by icerove** [comment]
when I run ```yarn test``` command, it will generate automatically. But I see you still do not have them in master branch, so I remove them. @vgrichina

**Comment by vgrichina** [comment]
> @icerove why did you remove `test.html` files? Are they auto-generated now?

nevermind, looks like "Test" still works and this is copied from `templates/`

**Pull request merged**